### ISSUE TITLE
Histogram of integers

### ISSUE THREAD

**Issue by jtrakk** [JuliaPlots/AlgebraOfGraphics.jl#129]
Currently integers aren't supported. That might actually be a good thing because it's ambiguous whether they should be treated as discrete or continuous, and it's easy for the user to convert. But it's worth thinking about.

```
data(DataFrame(x=sample(1:100, 10000))) * mapping(:x) * AlgebraOfGraphics.histogram(bins=200) |> draw

MethodError: no method matching floatrange(::Float64, ::Int64, ::Float64, ::Float64)
```

**Comment by piever** [comment]
Here I stick with what StatsBase does, so if they start supporting integer histograms, they will work here automatically, but before then, yes, you should just convert to floats.

**Comment by Moelf** [comment]
can we re-open this since StatsBase definitely supports integer bins

**Comment by piever** [comment]
No special casing is needed, now that StatsBase supports integer histograms, they just work in AoG.

**Comment by jtrakk** [comment]
Which version? 

```jl
data(DataFrame(x=rand(1:100, 10000))) * mapping(:x) * AlgebraOfGraphics.histogram(bins=200) |> draw
ERROR: MethodError: no method matching floatrange(::Float64, ::Int64, ::Int64, ::Float64)
```


  [cbdf2221] AlgebraOfGraphics v0.5.1

**Comment by piever** [comment]
Ups, my bad, the StatsBase function AoG calls to compute the edges still does not accept integers. Should be fixed by #261
